daphne_worker_test/wrangler.toml

```diff
@@ -40,7 +40,14 @@ DAP_TASKPROV_HPKE_COLLECTOR_CONFIG = """{
   "kem_id": "p256_hkdf_sha256",
   "kdf_id": "hkdf_sha256",
   "aead_id": "aes128_gcm",
-  "public_key": "047dab625e0d269abcc28c611bebf5a60987ddf7e23df0e0aa343e5774ad81a1d0160d9252b82b4b5c52354205f5ec945645cb79facff8d85c9c31b490cdf35466"
+  "public_key": { "value": [
```

Comment:
This isn't going to work for us unfortunately; we need to be able to encode the public key in hex. There should be some serde magic that can help: Search for `#[serde(with = "hex")]` in the repo. You may need to add a wrapper object: see https://serde.rs/container-attrs.html#try_from.

Reply:
done